src/test/java/com/ghilly/controller/CountryControllerTest.java

```diff
@@ -0,0 +1,72 @@
+package com.ghilly.controller;
+
+import com.ghilly.service.CountryServiceRest;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.mockito.Mockito.*;
+
+@SpringBootTest
+class CountryControllerTest {
+
+    private static final CountryServiceRest service = mock(CountryServiceRest.class);
+    private static final CountryController controller = new CountryController(service);
+    private static final int id = 100;
+    private static final String name = "USSR";
+
+
+    @Test
+    void createCountry() {
+        controller.create(name);
+
+        assertAll(
+                () -> verify(service).add(name),
+                () -> verifyNoMoreInteractions(service)
+        );
+    }
+
+    @Test
+    void getCountries() {
+        controller.getCountries();
+
+        assertAll(
+                () -> verify(service).receiveList(),
+                () -> verifyNoMoreInteractions(service)
+        );
+    }
+
+    @Test
+    void getCountry() {
+        controller.getCountry(id);
```

Comment:
method return value but you don't check it

Reply:
Done.